app/scripts/views/mixins/password-mixin.js

```diff
@@ -228,11 +229,20 @@ define(function (require, exports, module) {
 
     /**
      * Add the `highlight` class to the signup password helper. This is
-     * used to make the "A strong, unique..." text blue if either
+     * used to make the "Tips: Keep your password safe..." text blue if either
      * password field is focused.
+     *
+     * This also positions the password helper balloon on the correct password field.
+     * @param {Object} event Focus event
      */
-    highlightSignupPasswordHelper () {
+    highlightSignupPasswordHelper (event) {
       this.$(SELECTOR_SIGNUP_PASSWORD_HELPER).addClass('highlight');
+      // the code below moves the signup password helper between the two password fields
+      if (event && this.$(event.target).is('#password')) {
```

Comment:
If the passwords don't match, the balloon does not disappear when tabbing to other age field. Seems like it should?

<img width="660" alt="screen shot 2018-03-02 at 11 22 33 am" src="https://user-images.githubusercontent.com/1295288/36909516-2b20e812-1e0c-11e8-8254-bec43b6c0db1.png">

Reply:
> balloon does not disappear when tabbing to age field

@ryanfeeley should it?